third_party/nvfuser/csrc/lower_validation.cpp

```diff
@@ -425,12 +416,14 @@ class VectorizeValidator : public OptInDispatch {
 
     // Contiguity is based on rfactor domain.
     IterDomain* last_root_dim = nullptr;
+    size_t last_root_dim_pos;
```

Comment:
uninitialized `last_root_dim_pos`. If we have a tensor that's full size-1 (broadcasted), then we are `*tv->domain()->contiguity().at(last_root_dim_pos)`, that would be a segfault/UB

Reply:
I think this will be avoided by the `if (last_root_dim == nullptr)` before the `*tv->domain()->contiguity().at(last_root_dim_pos)`